File: lab-1-ralphp34-main/part1/prob2.c
//complete the C program
#include <stdio.h>

int main()
{
    int num;
    printf("Enter a number: \n");
    scanf("%d", &num);
    if (num <= 0)
    {
        printf("Houston, we have a problem\n");
    }

    printf("Lift off in...");
    for (int i = 0; i < num; i++)
    {
        printf("%d", num - i);
        printf("\n");

    }
        printf("\n");
        printf("        | \n");
        printf("       / \\ \n");
        printf("      / _ \\ \n");
        printf("     |.o '.| \n");
        printf("     |'._.'| \n");
        printf("     |     | \n");
        printf("   ,'|  |  |`. \n");
        printf("  /  |  |  |  \\ \n");
        printf("  |,-'--|--'-.| \n");

}

File: lab-1-ralphp34-main/part1/Prob2.java
import java.util.Scanner;

public class Prob2{

    public static void main(String args[]){

        Scanner sc = new Scanner(System.in);

	    
        System.out.println("Enter a number:");
        int a = sc.nextInt();
        System.out.println();


        if(a <= 0){
            //note printing to stderr!
            System.err.println("Houston, we have a problem");
            System.exit(1); //exit with e
        }
        
        System.out.println("Lift off in ...");
        for(int i=0;i<a;i++){
            System.out.println(a-i);
        }

        System.out.println();
        System.out.println("        | ");
        System.out.println("       / \\ ");
        System.out.println("      / _ \\ ");
        System.out.println("     |.o '.| ");
        System.out.println("     |'._.'| ");
        System.out.println("     |     | ");
        System.out.println("   ,'|  |  |`. ");
        System.out.println("  /  |  |  |  \\ ");
        System.out.println("  |,-'--|--'-.| ");
        
    }
}
